diff --git a/android/cpp-adapter.cpp b/android/cpp-adapter.cpp
index 74bfb293446a6ed4f6590acd84587787d9367b22..cff151036413294f3f6f30b4cfd38fa741dfabac 100644
--- a/android/cpp-adapter.cpp
+++ b/android/cpp-adapter.cpp
@@ -54,9 +54,4 @@ extern "C" JNIEXPORT void JNICALL Java_com_ocr_RNOcrModule_nativeInstall(JNIEnv
   _debugOutputDir = std::string(cDebugOuputDir);
   env->ReleaseStringUTFChars(assetDir, cAssetDir);
   env->ReleaseStringUTFChars(debugOutputDir, cDebugOuputDir);
-
-  // JavaVM *java_vm;
-  // jobject java_object;
-  // env->GetJavaVM(&java_vm);
-  // java_object = env->NewGlobalRef(thiz);
 }
diff --git a/android/src/main/java/com/ocr/RNOcrModule.kt b/android/src/main/java/com/ocr/RNOcrModule.kt
index b4eff4873058aba5d94febe4cc2a1bf7ac5b9ba8..26fb963d8e291e5e7310d26a793368ec473eded2 100644
--- a/android/src/main/java/com/ocr/RNOcrModule.kt
+++ b/android/src/main/java/com/ocr/RNOcrModule.kt
@@ -33,8 +33,6 @@ class RNOcrModule internal constructor(private val context: ReactApplicationCont
   override fun install() {
     try {
       val jsContext = getReactApplicationContext().getJavaScriptContextHolder()
-      // val jsCallInvokerHolder = context.catalystInstance.jsCallInvokerHolder as
-      // CallInvokerHolderImpl
 
       if (jsContext === null || jsContext.get() == 0L) {
         println("GutenOCR: JSI Runtime is not available in legacy chrome console")
diff --git a/node_modules/@gutenye/ocr-react-native/android/src/main/java/com/ocr/RNOcrSpec.kt b/android/src/main/java/com/ocr/RNOcrSpec.kt
new file mode 100644
index 0000000000000000000000000000000000000000..1feaf6f5a3db33f82fa23a27dbb5c6c4a65ffea1
--- /dev/null
+++ b/android/src/main/java/com/ocr/RNOcrSpec.kt
@@ -0,0 +1,10 @@
+package com.ocr
+
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+
+abstract class RNOcrSpec internal constructor(context: ReactApplicationContext) :
+    ReactContextBaseJavaModule(context) {
+
+  abstract fun install()
+}
