diff --git a/android/cpp-adapter.cpp b/android/cpp-adapter.cpp
index 39993f982a3713d8cc34076129853c7ec1543903..8d7535350ff3b229c891be6fbfdb5670abdc673c 100644
--- a/android/cpp-adapter.cpp
+++ b/android/cpp-adapter.cpp
@@ -54,9 +54,4 @@
   _debugOutputDir = std::string(cDebugOuputDir);
   env->ReleaseStringUTFChars(assetDir, cAssetDir);
   env->ReleaseStringUTFChars(debugOutputDir, cDebugOuputDir);
-
-  // JavaVM *java_vm;
-  // jobject java_object;
-  // env->GetJavaVM(&java_vm);
-  // java_object = env->NewGlobalRef(thiz);
 }
diff --git a/android/src/main/java/com/ocr/RNOcrModule.kt b/android/src/main/java/com/ocr/RNOcrModule.kt
index 57fd1d9bf9154df176e45d806d4b4151c8289bc3..0458e8ad5c0c7f79cc0c23a4b0617a68d1263561 100644
--- a/android/src/main/java/com/ocr/RNOcrModule.kt
+++ b/android/src/main/java/com/ocr/RNOcrModule.kt
@@ -33,8 +33,6 @@
   override fun install() {
     try {
       val jsContext = getReactApplicationContext().getJavaScriptContextHolder()
-      // val jsCallInvokerHolder = context.catalystInstance.jsCallInvokerHolder as
-      // CallInvokerHolderImpl
 
       if (jsContext === null || jsContext.get() == 0L) {
         println("GutenOCR: JSI Runtime is not available in legacy chrome console")
